frappe/public/js/frappe/list/bulk_operations.js

```diff
@@ -127,13 +129,18 @@ export default class BulkOperations {
 			args: {
 				doctype: "Letter Head",
 				fields: ["name", "is_default"],
+				filters: { disabled: 0 },
 				limit_page_length: 0,
 			},
 			async: false,
 			callback(r) {
 				if (r.message) {
 					r.message.forEach((letterhead) => {
-						letterhead_options.push(letterhead.name);
+						if (letterhead.is_default) {
+							letterhead_options.unshift(letterhead.name);
+						} else {
+							letterhead_options.push(letterhead.name);
+						}
 					});
```

Comment:
maybe we can push `No Letterhead` if no letter head is present in the system?

otherwise `letterheads[0]` above might cause errors?

Reply:
Line 126, `const letterhead_options = [__("No Letterhead")];` initializes the array with `No Letterhead` as it's first element. Ideally, `letterheads[0]` should always return something :smile: